### PR TITLE
js-ipfs 0.30 release blog post

### PR DESCRIPTION
**Pull request by alanshaw**

### CODE REVIEW

**Comment by alanshaw** [comment]
@diasdavid is good for merge?

File: content/post/40-js-ipfs-0.30.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
+date: 2018-07-09
+url: 40-js-ipfs-0-30
+title: js-ipfs 0.30.0 released
+author: Alan Shaw
+---
+
+Feature parity with go-ipfs just got a whole lot closer!
+
```
Comment (diasdavid) [inline content/post/40-js-ipfs-0.30.md]: Needs the fancy release tweet that people can RT

File: content/post/40-js-ipfs-0.30.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+---
+date: 2018-07-09
+url: 40-js-ipfs-0-30
```
Comment (diasdavid) [inline content/post/40-js-ipfs-0.30.md]: I believe this needs to have the `.md` too, right?

Comment (alanshaw) [reply content/post/40-js-ipfs-0.30.md]: Most of the others don't...

Comment (diasdavid) [reply content/post/40-js-ipfs-0.30.md]: nvm then :)

File: content/post/40-js-ipfs-0.30.md (hunk 1 above)
Comment (diasdavid) [reply content/post/40-js-ipfs-0.30.md]: Either one of 

- https://twitter.com/_alanshaw/status/1014990745508425728
- https://twitter.com/daviddias/status/1016364733107228673

**Comment by diasdavid** [comment]
The preview is not loading for me -- https://ipfs.io/ipfs/QmeMVUvZNkzkh6VFFHdvCc2dZETuCyqxd4XeRtUVcFcug9/ --, does that happen to anyone else?

**Comment by alanshaw** [comment]
Is work for me:

screen shot 2018-07-16 at 10 25 55

**Comment by diasdavid** [comment]
image

**Pull request merged**